Multicluster tutorial (#681)
kiali/kiali.io: content/en/docs/Tutorials/multicluster/02-Prerequisites.md

---
title: "Prerequisites"
description: "How to prepare for running the tutorial."
weight: 2
---

This tutorial is a walkthrough guide to install everything. For this reason, we will need:

* minikube
* istioctl
* helm

This tutorial was tested on:

* Minikube v1.30.1
* Istio v1.18.1
* Kiali v1.70

> Clusters are provided by minikube instances, but we can choose others instead, like OpenShift or just vanilla Kubernetes installations.

Reviewer: ```suggestion
Clusters are provided by minikube instances, but this tutorial should work on on any Kubernetes environment.
```

We will set up some environment variables for the following commands:

```
CLUSTER_EAST="east"
CLUSTER_WEST="west"

> ISTIO_DIR="absolute-path-to-istio-folder"

Reviewer: Assuming a bash shell, do these need `export` in front of them in order to be set for the following commands?
